src/content/extensions/multipleroutes/src/utils.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+var pn = chrome.privacy.network;
+var pi = null;
+
+// Handle the case when this is installed in pre-M48.
+if (pn.webRTCIPHandlingPolicy === undefined) {
+  chrome.privacy.IPHandlingPolicy = {};
+  chrome.privacy.IPHandlingPolicy.DEFAULT = 0;
+  chrome.privacy.IPHandlingPolicy.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES = 1;
+  chrome.privacy.IPHandlingPolicy.DEFAULT_PUBLIC_INTERFACE_ONLY = 2;
+  chrome.privacy.IPHandlingPolicy.DISABLE_NON_PROXIED_UDP = 3;
+}
+
+pi = chrome.privacy.IPHandlingPolicy;
+
+// Helper function to convert the parameters to policy synchronously.
+function convertToPolicy(allowMultiRoute, allowUdp, isInstall) {
+  if (allowMultiRoute) {
+    if (isInstall && pn.webRTCIPHandlingPolicy !== undefined) {
+      // If we're installing the extension, we'll default to a more private
+      // mode.
+      return pi.DEFAULT_PUBLIC_AND_PRIVATE_INTERFACES;
```

Comment:
This function serves as the purpose of mapping the old settings to the new world (even in older version of chrome). Since public_and_private_interfaces doesn't exist, we can't just return this out.